src/upgrade_step_change_moxa12XX_macros.py

```diff
@@ -0,0 +1,50 @@
+import traceback
```

Comment:
After running this script I successfully have an iocs.xml generated for a configuration containing the renamed MOXA12XX macros, however it does not seem to have an effect on the GUI configuration/component list. Is this behaviour expected? (even after a restart).

Reply:
Did you try restarting the IBEX server as well as the GUI? The new configuration should be loaded during IBEX startup.